wrapanapi/systems/openstack.py

```diff
@@ -376,6 +376,24 @@ def get_meta_value(self, key):
     def get_hardware_configuration(self):
         return {'ram': self.flavor.ram, 'cpu': self.flavor.vcpus}
 
+    @property
+    def attached_volumes(self):
+        """Find attached volumes of Openstack Instance/Server
+
+        Returns:
+            List of volume ids
+
+        Example:
+            .. code-block:: python
+
+               mgmt.get_vm(name='instance_name').attached_volumes
+        """
+        volumes_list = []
+        volumes = self.raw._info['os-extended-volumes:volumes_attached']
+        for volume in volumes:
```

Comment:
I would prefer to see a comprehension syntax here.

`return [v['id'] for v in self.raw._info['os-extended-volumes:volumes_attached']]`

Reply:
sure thing. :+1: